src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -231,6 +232,15 @@ public class JdbcSinkConnectorConfig {
             .withDefault(DefaultColumnNamingStrategy.class.getName())
             .withDescription("Name of the strategy class that implements the ColumnNamingStrategy interface.");
 
+    public static final Field POSTGRES_POSTGIS_SCHEMA_FIELD = Field.create(POSTGRES_POSTGIS_SCHEMA)
```

Comment:
I think that since for now we have only one configuration we can leave it as is and on the next one we can think to move to the dialects. I like the prefix `dialect` so for now, what do you think to rename the `database.postgres.postgis.schema` to `dialect.postgres.postgis.schema`?